apps/sms/test/unit/thread_ui_test.js

```diff
@@ -5225,7 +5239,9 @@ suite('thread_ui.js >', function() {
       Navigation.isCurrentPanel.withArgs('composer').returns(true);
 
       // Please, auto-answering stub, don't interfere with my test
-      this.sinon.stub(Contacts, 'findExact');
+      this.sinon.stub(Contacts, 'findExact').returns({
+        then: () => {}
```

Comment:
>  to do nothing

You mean you need kind-of-promise that never resolves? If so, maybe then ```new Promise(()=>{})```?

I don't really like that we change return signature for ```findExact``` that potentially can lead to a missed bug in the code